pkg/reconciler/taskrun/resources/image_exporter.go

```diff
@@ -37,9 +37,10 @@ func AddOutputImageDigestExporter(
 ) error {
 
 	output := []*v1alpha1.ImageResource{}
-	if len(tr.Spec.Outputs.Resources) > 0 {
-		for _, trb := range tr.Spec.Outputs.Resources {
-			boundResource, err := getBoundResource(trb.Name, tr.Spec.Outputs.Resources)
+	fmt.Println(tr.Spec.Resources)
```

Comment:
Is this a debugging log or intended to be removed?  Same for line 75

Reply:
:upside_down_face: yeah I need to clean some stuff up :stuck_out_tongue_closed_eyes: 
There is a few `logger.…` at some random places too :wink: :angel: